### PR TITLE
siril: 0.99.6 -> 0.99.8.1

### PR DESCRIPTION
**Pull request by hjones2199**
###### Motivation for this change
Update siril to newest version and add dependencies for new feature support.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 114271` at 3d1486ff run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/bc55c224ffffec63877c51bb65db03af">siril</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/science/astronomy/siril/default.nix:41:5:
```
   |
41 |     license = licenses.gpl3;
   |     ^
```
</li>
  </ul>
</details>

**Comment by hjones2199** [comment]
Last force-push changes license to gpl3Plus. I checked https://gitlab.com/free-astro/siril/-/blob/master/LICENSE.md and it includes the "any later version" clause

File: pkgs/applications/science/astronomy/siril/default.nix [hunk 1]
```diff
@@ -1,27 +1,27 @@
-{ lib, stdenv, fetchFromGitLab, fetchFromGitHub, pkg-config, meson, ninja,
-  git, criterion, wrapGAppsHook, gtk3, libconfig, gnuplot, opencv,
-  fftwFloat, cfitsio, gsl, exiv2, curl, librtprocess, ffmpeg,
+{ lib, stdenv, fetchFromGitLab, pkg-config, meson, ninja, wrapGAppsHook,
+  git, criterion, gtk3, libconfig, gnuplot, opencv, json-glib,
```
Comment (SuperSandro2000) [inline pkgs/applications/science/astronomy/siril/default.nix:2]:
```suggestion
, git, criterion, gtk3, libconfig, gnuplot, opencv, json-glib
```
Please move the , to the beginning of the line.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/science/astronomy/siril/default.nix (hunk 1 above)
Comment (hjones2199) [reply pkgs/applications/science/astronomy/siril/default.nix]: Thanks for the review. I've made the requested change, marking as resolved.

**Pull request merged**